bin/data_to_model_score.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+
+
+def score_submission(predictions_path: str, status: str) -> dict:
+    """Determine the score of a submission. This is a placeholder function.
+
+    Args:
+        predictions_path (str): path to the predictions file
```

Comment:
nit: I think since you used type-hints in the definition, you don't need to specify the data type in the arguments section.